app/src/main/java/com/nervousfish/nervousfish/activities/MainActivity.java

```diff
@@ -137,47 +122,46 @@ private void fillDatabaseWithDemoData() throws IOException {
         database.addContact(e);
     }
 
-}
-
-/**
- * An Adapter which converts a list with contacts into List entries.
- */
-final class ContactListAdapter extends ArrayAdapter<Contact> {
-
-    /**
-     * Create and initialize a ContactListAdapter.
-     *
-     * @param context  the Context where the ListView is created
-     * @param contacts the list with contacts
-     */
-    ContactListAdapter(final Context context, final List<Contact> contacts) {
-        super(context, 0, contacts);
-    }
-
     /**
-     * {@inheritDoc}
+     * An Adapter which converts a list with contacts into List entries.
      */
-    @NonNull
-    @Override
-    public View getView(final int position, final View convertView, @NonNull final ViewGroup parent) {
-        View v = convertView;
-
-        if (v == null) {
-            final LayoutInflater vi = LayoutInflater.from(getContext());
-            v = vi.inflate(R.layout.contact_list_entry, null);
+    private static final class ContactListAdapter extends ArrayAdapter<Contact> {
```

Comment:
Yes, because Findbugs warns you that ContactListAdapter should be a private static inner class instead of an non-static outer class